docs/static/monitoring/monitoring-internal-legacy.asciidoc

```diff
@@ -155,8 +155,8 @@ xpack.monitoring.elasticsearch.sniffing: false
 . Restart your Logstash nodes.
 
 . To verify your monitoring configuration, point your web browser at your {kib}
-host, and select **Monitoring** from the side navigation. Metrics reported from
-your Logstash nodes should be visible in the Logstash section. When security is
+host, and select **Stack Monitoring** from the side navigation. If this is an initial setup, choose *Or, set up with self monitoring* option from the UI and click *Turn on monitoring* button.
+Metrics reported from your Logstash nodes should be visible in the Logstash section. When security is
```

Comment:
```suggestion
host, and select **Stack Monitoring** from the side navigation. 
If this is an initial setup, select *set up with self monitoring* and click *Turn on monitoring*.
Metrics reported from your Logstash nodes should be visible in the Logstash section. 
When security is
```

Reply:
Let's start each new sentence on a new line. I made a few suggestions for writing about the UI.  I took out the "Or" because the text reads better and users are not likely to be confused without it.